tests/manage/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -111,50 +116,50 @@ def test_non_ocs_taint_and_tolerations(self):
                 kind=constants.SUBSCRIPTION,
             )
             sub_obj.patch(params=param, format_type="merge")
+            logger.info(f"Successfully added toleration to {sub}")
+
+        # Add tolerations to the configmap rook-ceph-operator-config
+        configmap_obj = ocp.OCP(
+            kind=constants.CONFIGMAP,
+            namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            resource_name=constants.ROOK_OPERATOR_CONFIGMAP,
+        )
+
+        params = (
```

Comment:
does this work on 4.11 too? Please test the test on 4.11 as well

Reply:
yes I ran it on 4.11 locally and it is working well